src/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -637,7 +637,7 @@ public boolean match(final Unit transport) {
     capableTransports.removeAll(alliedTransports);
     // First, load capable transports
     final Map<Unit, Unit> unitsToCapableTransports =
-        TransportUtils.mapTransports(route, availableUnits, capableTransports);
```

Comment:
These 3 method updates are to hook up the new transport loading method that finds minimum number of needed transports.